.gitignore

```diff
@@ -30,3 +30,4 @@ build/
 configs
 !configs/config.toml.SAMPLE
 external-contracts.js-e
+include/
```

Comment:
Is it still needed given that the submodule lives in `staker-rewards/merkle-distributor`?

Reply:
No, just a leftover, I'll remove it